frame/composable-traits/src/dex.rs

```diff
@@ -32,6 +32,9 @@ pub trait Amm {
 
 	fn lp_token(pool_id: Self::PoolId) -> Result<Self::AssetId, DispatchError>;
 
+	/// Returns the xToken given to users when they stake
+	fn x_token(pool_id: Self::PoolId) -> Result<Self::AssetId, DispatchError>;
```

Comment:
Why is this needed in the dex trait?

Reply:
It doesn't, tho I might consider moving the new functions to their own traits so they can be tested more easily
e065b5f9ac97e99d3f08d2cc5189c0d1d3c0b2e4